src/Microsoft.ML.Mkl.Components/SymSgdClassificationTrainer.cs

```diff
@@ -48,6 +48,7 @@ namespace Microsoft.ML.Trainers
     /// | Is normalization required? | Yes |
     /// | Is caching required? | No |
     /// | Required NuGet in addition to Microsoft.ML |Microsoft.ML.Mkl.Components |
+    ///
     /// ### Training Algorithm Details
     /// The symbolic SGD is a classification algorithm that makes its predictions by finding a separating hyperplane.
     /// For instance, with feature values $f0, f1,..., f_{D-1}$, the prediction is given by determining what side of the hyperplane the point falls into.
```

Comment:
This description doesn't directly reflect the true prediction rule: sign of score is the predicted label. If score > 0, predicted label is true and false otherwise.

Reply:
That is the same as the sign of the features' weighted sum, i.e. $\sum_{i = 0}^{D-1} (w_i * f_i)$, where $w_0, w_1,..., w_{D-1}$ are the weights computed by the algorithm.

---
In reply to: [277147921](https://github.com/dotnet/machinelearning/pull/3396#discussion_r277147921) [](ancestors = 277147921)